doc/tutorial/examples.txt

```diff
@@ -500,8 +500,8 @@ It will be used repeatedly.
   training_steps = 10000
   
   # Declare Theano symbolic variables
-  x = T.matrix("x")
-  y = T.vector("y")
+  x = T.dmatrix("x")
+  y = T.dvector("y")
```

Comment:
If we want to make that example work with float32, it would be better to get the code from the test instead:
https://github.com/nouiz/Theano/blob/mixed/theano/tests/test_tutorial.py#L727-L776